mkosi/mkosi.images/build/mkosi.conf.d/arch/mkosi.build.chroot

```diff
@@ -22,7 +22,7 @@ if ((LLVM)) && [[ -n "$SANITIZERS" ]]; then
     MKOSI_LDFLAGS="$MKOSI_LDFLAGS -Wl,-rpath=$(realpath "$(clang --print-runtime-dir)")"
 fi
 
-MKOSI_MESON_OPTIONS="-D mode=developer -D vcs-tag=${VCS_TAG:-true} -D b_sanitize=${SANITIZERS:-none}"
+MKOSI_MESON_OPTIONS="-D mode=developer -D vcs-tag=${VCS_TAG:-true} -D b_sanitize=${SANITIZERS:-none} -Dtime-epoch=1744207869"
```

Comment:
Hmm, that seems iffy. This is now hardcoded to `Wed Apr  9 14:11:09`… Is it supposed to stay like that forever?

Reply:
Until it causes issues, yeah. These builds are for development, so I don't think it'll be a big problem. Every time this value changes a full rebuild happens and I couldn't find anything satisfactory besides hardcoding a value.